src/constants/examples.constant.ts

```diff
@@ -1,6 +1,10 @@
 import { ENUMS } from './enum.constant';
+import { MAX_FILE_SIZE_BYTES } from './max-file-size-bytes.constant';
 
 export const EXAMPLES = {
   SITE_ID: '12345678',
   SITE_STATUS_CODE: ENUMS.SITE_STATUSES.CREATED,
+  FILE_SIZE: MAX_FILE_SIZE_BYTES,
```

Comment:
`FILE_SIZE_BYTES`?

Reply:
Personally I think it's most useful if the name of the example matches the name of the field it is an example for, which is `fileSize`